usaspending_api/wsgi.py

```diff
@@ -9,8 +9,88 @@
 
 import os
 
+from opentelemetry.instrumentation.wsgi import OpenTelemetryMiddleware
 from django.core.wsgi import get_wsgi_application
+from usaspending_api.common.logging import configure_logging
+from opentelemetry import trace
+from opentelemetry.instrumentation.django import DjangoInstrumentor
+
+
+def request_hook(span, environ):
+
+    if span and span.is_recording():
+        headers_to_capture = [
```

Comment:
Is it important that the request and response are both different casing and style (snake case vs kebab case)? Should this be a constant defined at the top of this file?

Reply:
I fixed the inconsistency, now the headers will have the same lowercase, kebab-case convention for both the request hook and the response hook.